pkg/config/endpoint.go

```diff
@@ -171,6 +171,24 @@ type userInfo struct {
 	Subscription int
 }
 
+func (u *userInfo) UnmarshalJSON(raw []byte) error {
+	v := struct {
+		Age          uint8  `json:"age"`
+		Registered   string `json:"registered"`
```

Comment:
Totally missed this, but `Registered` should be of `time.Time` so we can better handle it down the road. Same for the domain struct we pass around internally. This way we have an easier time later when enabling date comparisons like before and after.